JS/Yeva/BooksStore/src/Book.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import data from './books.json'
+
+const count_form = document.querySelector('#count-form');
+
+count_form.addEventListener('submit', function (event) {
```

Comment:
Works fine. Not bad but could you make it better and do not do the same thing several time